Library/Formula/openshift-cli.rb

```diff
@@ -0,0 +1,33 @@
+class OpenshiftCli < Formula
+  desc "OpenShift command-line interface tools"
+  homepage "https://www.openshift.com/"
+  url "https://github.com/openshift/origin.git",
+    :tag => "v1.1.1",
+    :revision => "e1d9873c1d5711b83fd3dd7eefe83a88ceb92c08"
+
+  head "https://github.com/openshift/origin.git"
+
+  depends_on "go" => :build
+
+  def install
+    # this is necessary to avoid having the version marked as dirty
+    (buildpath/".git/info/exclude").atomic_write <<-EOS.undent
```

Comment:
Given how compact this is you probably don't need the whole `EOS.undent` block:

```ruby
(buildpath/".git/info/exclude").atomic_write "/.brew_home"
```
Should work.